lib/doctest.mjs

```diff
@@ -0,0 +1,62 @@
+import {readFile, writeFile, unlink} from 'fs';
+import {resolve} from 'path';
+import {promisify} from 'util';
+
+import common from './common';
+import doctest from './doctest.js';
```

Comment:
The file extension is required here, I assume.

Reply:
Yes, because Node favours the `.mjs` file extension by default when running esm modules.